in-progress/7520-mainnet-overview.md

```diff
@@ -0,0 +1,349 @@
+
+Deployments will have a mechanism for forced inclusions of transactions in the canonical chain.
+
+### Open Questions
+- What are the requirements for a forced inclusion?
+- Is there any DoS risk?
+- Is this influenced by the sequencer selection process?
+
+## The Aztec Token
+
+The Aztec Token (AZT) is an ERC20 token that is used to pay for transaction fees on the Aztec Network.
+
+It is also used on L1 as part of the validator selection process.
+
+Protocol incentives are paid out in AZT.
+
+AZT is bridged from L1 to L2 using a trusted bridge.
+
+Once on L2, AZT is exclusively to used to pay transaction fees; it cannot be transferred back to L1 or transferred to other users.
+
+## Compliance
+
+### Open Questions
+- What are the compliance requirements for the Aztec Network?
+
+
+## Aztec Labs Node
+
+Aztec Labs will provide a reference implementation of the Aztec Node, which will be used to run the Aztec Network.
+
+It will have 3 primary modes of operation:
+- Proposer/Validator: responsible for proposing new blocks and validating them
+- Prover: responsible for orchestrating the creation of various proofs
+- Full Node: follows along, responsible for propagating transactions and blocks
+
+The node will have a web interface for monitoring key metrics.
+
+
+## Chains, slots, and epochs
+
+There are two chains in the Aztec Network:
+- The Pending Chain
+- The Proven Chain
+- The Finalized Chain
+
+All three chains are effectively managed by the Aztec Node and the L1 contracts, and have different guarantees.
+
+Time is divided into slots, which are grouped into epochs.
+
+Each slot has a proposer, who is responsible for proposing a block of transactions.
+
+Each epoch has a set of validators, who are responsible for validating the blocks proposed by the proposers.
+
+### Open Questions
+- Would a "pure L2" chain help us achieve our goals?
+- How long should a slot be?
+- How long should an epoch be?
+
+
+## The Pending Chain
+
+The purpose of the pending chain is to reduce the perceived latency of transactions: it allows clients to observe transactions that have been proposed, but the proof has not yet been made available.
+
+The proposer for a slot produces a list of transaction objects either by
+- selecting them from the L2 mempool,
+- or receiving a list from a builder.
+
+The proposer gossips to the validators:
+- A signature showing it is the current proposer
+- The list of transaction objects
+
+Validators check that the proposer is the current proposer.
+They then build an L2 block by executing the transaction objects, and sign the resulting L2 block header.
+
+Once the proposer has seen enough signatures, they can submit the L2 block to L1.
+
+The proposer submits the block header as calldata to a function on the rollup contract dedicated to advancing the pending chain.
+
+### Open Questions
+- How many signatures are required?
+- Does the need to execute the transaction objects create too much of a burden on validators?
```

Comment:
I'm asking about the B- case where the are no longer required to execute, so what their signature means changes to "I saw the Tx, and if it is valid, then it can be proven". So they shouldn't be slashed if it is invalid in that case, but I don't understand all the ramifications there.

So this question is an indirect question of "is the only reason we don't want validators to re-execute because we're concerned that simulation will demand powerful machines that will reduce our validator set?", and a follup question of "if the demands of simulation aren't that high, why would we not just have them re-execute? because it seems like having them sign over just seeing data is very risky".

Reply:
In that case is there only one header per epoch?